test/doctool/test-doctool-html.js

```diff
@@ -140,21 +139,20 @@ const versions = [
   { num: '0.12.x' },
   { num: '0.10.x' }];
 
-testData.forEach(({ file, html }) => {
+testData.forEach(async ({ file, html }) => {
   // Normalize expected data by stripping whitespace.
   const expected = html.replace(spaces, '');
 
-  readFile(file, 'utf8', common.mustCall(async (err, input) => {
-    assert.ifError(err);
-    const output = toHTML({ input: input,
-                            filename: 'foo',
-                            nodeVersion: process.version,
-                            versions: versions });
+  const input = await readFile(file, 'utf8');
```

Comment:
No async API needed.

```suggestion
  const input = await readFileSync(file, 'utf8');
```

Reply:
```suggestion
  const input = readFileSync(file, 'utf8');
```